openapi/code_samples/python/api_v1_users_{id}/patch.py

```
@@ -1,18 +1,11 @@
     user_profile=UserProfile(
         first_name="John",
-        last_name="Smith"
+        last_name="Smith",
     ),
-    metadata={"department": "sales"}
+    metadata={"department": "sales"},
 )
-scalekit_client.users.update_user(organization_id="org_123", 
-  user=update_user)
\ No newline at end of file
+response = scalekit_client.users.update_user("usr_123", update_user)
```

Ruff (0.15.21)

[error] Undefined name `scalekit_client`

(F821)